packages/core-components/src/components/LogViewer/RealLogViewer.tsx

```diff
@@ -81,70 +84,103 @@ export function RealLogViewer(props: RealLogViewerProps) {
     selection.setSelection(line, event.shiftKey);
   };
 
+  function setRowHeight(index: number, size: number) {
+    if (shouldTextWrap && listInstance) {
+      (listInstance as VariableSizeList<AnsiLine[]>).resetAfterIndex(0);
+      // lineNumber is 1-based but index is 0-based
+      heights.current = { ...heights.current, [index - 1]: size };
```

Comment:
```suggestion
      heights.current[index - 1] = size;
```

that spread is going to get expensive to perform over and over for every single line, for large files